requirements.txt

```diff
@@ -1 +1,2 @@
 numpy>=1.15
+scipy>=0.13.3
```

Comment:
claudio, we have been to mantain mathphys as slim as possible over these years. we plain to remove scipy even from siriuspy. why do you need to include scipy as a dependency of the basic mathphys package? scipy is a relatively large package.

Reply:
Indeed, this seems to be a mistake. Thanks!

Reply:
by the way, we should move this repo to lnls-sirius organization!